deps/dicer/lib/HeaderParser.js

```diff
@@ -72,39 +72,37 @@ HeaderParser.prototype._parseHeader = function() {
   if (this.npairs === this.maxHeaderPairs)
     return;
 
-  var lines = this.buffer.split(RE_CRLF), len = lines.length, m, h,
-      modded = false;
+  const lines = this.buffer.split(RE_CRLF), 
+    len = lines.length;
+  let m, h;
 
-  for (var i = 0; i < len; ++i) {
+  for (let i = 0; i < len; ++i) {
```

Comment:
`var` actually is more performant here, because with `let` you will create a new variable instance on each iteration. This was observed in other fastify libraries and `var` is preferable for iteration in performance-critical scenarios.

Reply:
Oha! New information. Will change